spec/lib/bank_account/savings_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'bank_account'
+
+describe Bank::SavingsAccount do
+  context "attributes" do
```

Comment:
It looks like all the tests in this file are wrapped in this `context` block. You could probably get rid of it, or just wrap the ones related to attributes in it. I tend to use `context` blocks for setting up edge case situations, rather than for testing the happy path situation.